### ISSUE TITLE
Swift Codegen Not Respecting --mergeInFieldsFromFragmentSpreads Flag

### ISSUE THREAD

**Issue by alex-king-ck** [apollographql/apollo-cli#537]
Using CLI version 0.2.0 (The latest version supported by apollo-ios)
The same issue is present when using 0.6.0 (latest current version)

We're trying to upgrade our version of apollo-ios. This means updating our version of the CLI (coming from apollo-codegen v0.17). Generating our API.swift file with all of our queries produced a 180MB file. We had seen this previously and the resolution was to disable merging fields from fragment spreads. However, the output of apollo-cli 0.2.0 doesn't change when we use the `--mergeInFieldsFromFragmentSpreads` flag.

Here is a small sample query that can be used to illustrate the issue:
```
query Hub($useForce: Boolean) {
    hubsInfo(useForce: $useForce) {
        hubs {
            ...hubInfo
        }
    }
}

fragment hubInfo on Hub {
    discriminator
    label {
    ...formattedTextInfo
    }
}


fragment formattedTextInfo on FormattedText {
    spans {
        text
        format {
            italic
            strong
        }
    }
}
```

Generating using apollo-cli v0.6.0 with and without `--mergeInFieldsFromFragmentSpreads` produced the following output:
[API.swift.txt](https://github.com/apollographql/apollo-cli/files/2271589/API.swift.txt)

Here is the expected output, generated with apollo-codegen v0.17: 
[API.swift.txt](https://github.com/apollographql/apollo-cli/files/2271595/API.swift.txt)

**Comment by chrisco314** [comment]
@martijnwalraven 

@alex-king-ck, I previously had this flag set as  --merge-in-fields-from-fragment-spreads false, after the --output API.swift, to avoid the code bloat that you speak of.  I feel like the syntax has changed, can you tell me what your current syntax is?  I get an error if I include it at all now.

Previously I had something like: 
`--schema service/schema.json --output service/API.swift --merge-in-fields-from-fragment-spreads false`

but it appears that the flag name has changed to camelcase, and that output has been removed.  Have you figured out of you can set this flag to false or is one only supposed to use it for the positive?

Thanks,

Chris

**Comment by martijnwalraven** [comment]
This should be fixed in Apollo iOS 0.9.3 (which depends on the Apollo CLI 1.8.x).
